From 61bc2bc7d428cafe45f6d9ab40e6c05262d307e9 Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Tue, 30 Oct 2012 01:11:14 +0100
Subject: [PATCH 159/278] x86util: Add cpuflags_mmxext alias for cpuflags_mmx2

"mmxext" is a more sensible name and more common in outside projects.
---
 libavutil/x86/x86util.asm |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/libavutil/x86/x86util.asm b/libavutil/x86/x86util.asm
index 4c6f4c6..761cea0 100644
--- a/libavutil/x86/x86util.asm
+++ b/libavutil/x86/x86util.asm
@@ -23,6 +23,8 @@
 ;* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
 ;******************************************************************************
 
+%define cpuflags_mmxext cpuflags_mmx2
+
 %include "libavutil/x86/x86inc.asm"
 
 %macro SBUTTERFLY 4
-- 
1.7.9.5

